scripts/load_cql.py

```diff
@@ -158,6 +163,17 @@ def setup_connection(reactor, args):
         args.keyspace)
 
 
+def webhook_index(reactor, args):
+    """
+    Show webhook indexes that is not there table connection
+    """
+    store = CassScalingGroupCollection(None, None)
+    eff = store.get_webhook_index_only()
+    conn = setup_connection(reactor, args)
+    return perform(get_cql_dispatcher(reactor, conn), eff).addCallback(
```

Comment:
Nonblocking - in case there are any errors, so that we can see this before we continue, do we want to do `addBoth(pprint)`